platform.js

```diff
@@ -102,8 +102,9 @@ module.exports = {
       detached: true
     }).unref();
   },
-  spawn(command, args, timeout) {
-    log.debug("executing " + command + " with " + args);
+  spawn(command, args, timeout, tries) {
+    if (tries === undefined) {tries = 1;}
+    log.debug("try " + tries + " executing " + command + " with " + args);
```

Comment:
@tejohnso if args is a string, it might be worthwhile to do something like `args.join(" ")`